### PR TITLE
Edit for GitLab section, fix formatting for Code Manager.

### PR DESCRIPTION
**Pull request by misseuropa**
If this is still welcome, great, if not let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,39 +42,39 @@ When you finish the instructions below, you will have the beginning of a best pr
 
 ### GitLab
 
-1. On a new server, install GitLab
+1. On a new server, install GitLab CE.
  - https://about.gitlab.com/downloads/
 
-2. After GitLab is installed you may sign into the web UI with the `root` user and password `5iveL!fe`
+2. After GitLab is installed, sign into the web UI with the user `root`
```
Comment (misseuropa) [inline README.md:48]: They make you reset the password, you can't use the 5iveL!fe one.

File: README.md [hunk 2]
```diff
@@ -42,39 +42,39 @@ When you finish the instructions below, you will have the beginning of a best pr
...
+7. In the GitLab UI, add your user to the `puppet` group.
+ - You must give your user at least master permissions to complete the following steps.
  - Read more about permissions:
     - https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/permissions/permissions.md
 
-8. In the GitLab UI, create a project called `control-repo` and set its Namespace to the `puppet` group
+8. In the GitLab UI, create a project called `control-repo` and set its Namespace to the `puppet` group.
 
-10. On your laptop, clone this PuppetLabs-RampUpProgram control repo
+10. On your laptop, clone this PuppetLabs-RampUpProgram control repo.
```
Comment (misseuropa) [inline README.md:67]: You could choose to split setting up GitLab server from cloning the repo. No big deal right now, but easier later on if/when you add other Git hosting services.

**Comment by npwalker** [comment]
https://github.com/PuppetLabs-RampUpProgram/control-repo/commit/3cb199565e252d4d9930a126155a90afe4205bcb

File: README.md (hunk 2 above)
Comment (npwalker) [inline README.md:67]: Agreed, we'll cross that bridge when we get to it.

File: README.md (hunk 1 above)
Comment (npwalker) [inline README.md:48]: I added a note about that behavior.

**Pull request closed without merging**